lib/src/client.dart

```diff
@@ -140,3 +141,48 @@ abstract class Client {
   /// do so can cause the Dart process to hang.
   void close();
 }
+
+const _clientToken = Object();
+
+/// Return the [Client] for the current [Zone], if one has been set.
```

Comment:
Document getters using noun phrases. I'd just remove the "Return". (Nit: Which, if it had stayed, should be active form "Returns".)